src/helpers/dateTime.ts

```diff
@@ -0,0 +1,30 @@
+import moment from 'moment'
+const months = {
+  0: "Jan",
+  1: "Feb",
+  2: "Mar",
+  3: "Apr",
+  4: "May",
+  5: "Jun",
+  6: "Jul",
+  7: "Aug",
+  8: "Sep",
+  9: "Oct",
+  10: "Nov",
+  11: "Dec",
+};
+
+export const convertStrToDate = (str: string | number) => {
```

Comment:
isn' this method formatting the string the way I described above?

Reply:
I removed this function as we only use calcTimeElapsed function